README.md

```diff
@@ -16,9 +16,12 @@ To verify your install, run this command:
 ```
 hugo version
 ```
```

Comment:
Modify step 1 to look like this;
![Screenshot (3)](https://user-images.githubusercontent.com/49279842/97856986-21474a80-1d0e-11eb-9e10-29442e9e24ab.png)

Reply:
you can use `visit this link` instead  of `go to this link`